### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -13,72 +13,128 @@
         .comp-lib-hr, .comp-lib-h1, .comp-lib-h2 {
             margin: 1rem;
         }
+        .component-holder {
+            display: flex;
+            flex-direction: row;
```
Comment (iRohitGaur) [inline components/alert/alert.html:18]: by default the direction is row. no need for this line

File: components/alert/alert.html [hunk 2]
```diff
@@ -13,72 +13,128 @@
...
+        
+                <div class="alert__content">
+                    <div class="alert__text alert__text--success">
+                        <h3 class="alert__title">Success</h3>
+                        <p class="alert__description dui-util-txt-sm">This is a success alert — <strong>check it out!</strong></p>
+                    </div>
+                </div>
+            </div>
+            <!-- Alert Component Ends -- Success -->
+            <div class="code_snippets_div">
```
Comment (iRohitGaur) [inline components/alert/alert.html:53]: if you are following a naming convention then use the same everywhere for consistency

**Review by iRohitGaur — APPROVED** [review]

File: library/card/card.css [hunk 3]
```diff
@@ -24,11 +22,9 @@
 }
 
 .card__info {
```
Comment (zoef-yt) [inline library/card/card.css:24]: can be removed if not in use

File: library/card/card.css [hunk 4]
```diff
@@ -24,11 +22,9 @@
...
-    padding: 0.8rem 0.4rem 0.4rem 0.4rem;
 }
 
 .card__primary-text{
-    font-size: 2rem;
     color: var(--light-theme-contrast);
 }
```
Comment (zoef-yt) [inline library/card/card.css:29]: spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: library/card/card.css [hunk 5]
```diff
@@ -24,11 +22,9 @@
...
 .card__secondary-text{
```
Comment (purvasharmadev) [inline library/card/card.css:30]: Should be removed if not used.

**Comment by itsmebipul00** [comment]
some utility classes have very large names other than that rohit has covered alot, utilty for flex could also be used since it is repeatedly used, and great refactor!!

**Pull request merged**